### ISSUE TITLE
Finish buildout of platform folder

### ISSUE THREAD

**Issue by rroueche** [department-of-veterans-affairs/va.gov-team#1240]

**Comment by rroueche** [comment]
### Goal
Any VFS Team can find the appropriate resources for how to engage w/VSP or more globally, how to build on VA.gov, in the platform folder in va.gov-team repo.

### Description
We have started auditing the legacy va.gov-team content from /Administrative, /Platform, and /Practice Areas sections for top level hierarchies. We need to complete that audit, down to the sub-folder levels, and determine where that content should be moved to and if that impacts the current proposed ia.

### Definition of Done
- [ ] va.gov-team audit complete
- [ ] vfs-teams audit complete
- [ ] Per audit of current va.gov-team content, we know where things should be rehomed and have updated the ia per any findings
- [ ] ^ is documented and shared across VSP and VFS
- [ ] Have full platform skeleton in va.gov-team repo

**Comment by mnorthuis** [comment]
- va.gov-team repo has been audited and each file now has a planned new home
- I have been talking with content owners as I've worked through this audit to confirm where documentation should go, these conversations do not seem pertinent to VFS teams
- the `platform` folder structure is built out within the repo

We will be moving the actual files in the beginning sprint 8.  This work should not include the vfs-teams repo. 

Closing this issue as work is complete. Sprint 8 will include an issue for moving the files.
